File: src/main/java/bank/place/PlaceService.java
package bank.place;

import java.util.List;

public interface PlaceService {
	public List<Place> findAll();

	public Place save(Place place);

	public Place findOne(Long id);
	
	public void delete(Long id);
	
	public List<Place> search(Place place);

	public Place findByName(String place);
}

File: src/main/resources/static/place/placeService.js
var services = angular.module('place.services', [ 'ngResource' ]);

services.service('placeService', [ '$http', function($http) {
	
	this.findAllCountries = function(){
		return $http.get("/country");
	}
	this.findAll = function() {
		return $http.get("/place");
	}
	
	this.save = function(place) {
		return $http.post("/place",place);
	}
	
	this.next = function(id) {
		return $http.get("/place/nextPlaces/" + id);
	}
	
	this.delete = function(place){
		return $http.get("/place/deletePlace/" + place);
	}
	this.search = function(entity){
		return $http.post('/place/search',entity);
	}
	
	this.update = function(place){
		return $http.put("/place/update", place);
	}
}]);